plugins/profiles.koplugin/main.lua

```diff
@@ -268,6 +283,77 @@ function Profiles:editProfileName(editCallback, old_name)
     name_input:onShowKeyboard()
 end
 
+function Profiles:getProfileFromCurrentDocument(new_name)
```

Comment:
I'd possibly add a cross-reference to this in ConfigDialog so that it doesn't get forgotten whenever someone modifies relevant ConfigDialog settings ;).

Reply:
In the same vein, I imagine there's too much unrelated stuff in configurable to just toss the whole thing without having to keep a whitelist?

Reply:
Yes, and some of them do not have a corresponding action in Dispatcher and cannot be set via a profile (gesture).
Besides, the whitelists are used to set an order of profile actions, useful when reviewing the profiles as QuickMenus.